### PR TITLE
Commiting Filter library changes in new branch

### PR DESCRIPTION
**Pull request by pkharat**
@njagadish

### CODE REVIEW

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (pkharat) [inline .cache/v/cache/lastfailed]: Suresh Please remove this file

File: gbpservice/neutron/tests/unit/nsf/configurator/lib/filter_base.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
+import unittest
+
```
Comment (njagadish) [inline gbpservice/neutron/tests/unit/nsf/configurator/lib/filter_base.py]: Need to remove all the files under nsf directory.

File: gbpservice/neutron/tests/unit/nsf/configurator/lib/filter_base.py [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+
+class BaseTestCase(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(BaseTestCase, self).__init__(*args, **kwargs)
```
Comment (njagadish) [inline gbpservice/neutron/tests/unit/nsf/configurator/lib/filter_base.py]: Need to change the path of unit tests to "gbpservice/nfp/test/unit/configurator/lib" directory.

**Comment by njagadish** [comment]
Merging this PR to facilitate integration testing. Unit testing of this code is done. Code review is pending.

**Pull request merged**